app/assets/src/components/views/discovery/mapping/BaseMap.jsx

```diff
@@ -68,6 +68,7 @@ class BaseMap extends React.Component {
           onViewportChange={this.updateViewport}
           mapStyle={styleURL}
           onLoad={this.setCompactAttribution}
+          style={{ position: "absolute" }} // Fixes div in Safari
```

Comment:
can you use css instead?

Reply:
in this case I think it was better to just pass it through (otherwise a lot of CSS overriding to do for the markers as well) 
![Screen Shot 2019-06-10 at 12 54 29 PM](https://user-images.githubusercontent.com/5652739/59222735-099e7d80-8b7f-11e9-96c4-4a945b0b15c8.png)